tests/Tests/500__jupyterhub/minimal-pytorch-test.robot

```diff
@@ -34,6 +34,19 @@ PyTorch Image Workload Test
     Capture Page Screenshot
     JupyterLab Code Cell Error Output Should Not Be Visible
 
+Tensorboard Workload Test
```

Comment:
why `Workload`? sounds like a load testing

Reply:
In order to show something in Tensorboard you need to run a workload, albeit small :) I don't see a problem with the name

Reply:
Maybe a more self-explanatory name would have been sth like `Verify Tensorboard Is Accessible` or similar.

Reply:
Done